src/bulkExport/bulkExportResultsUrlGenerator.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+export interface BulkExportResultsUrlGenerator {
+    getUrls(options: { bucket: string; keys: string[] }): Promise<{ requiresAccessToken: boolean; urls: string[] }>;
```

Comment:
Since we currently require export to be put in S3 thoughts on making the params more explicit? 
```suggestion
    getUrls(options: { exportBucket: string; s3Keys: string[] }): Promise<{ requiresAccessToken: boolean; urls: string[] }>;
```

Reply:
sounds good. I just renamed them